doc/mgr/smb.rst

```diff
@@ -608,8 +608,20 @@ cephfs
         subvolume field will automatically be split into
         ``<subvolumegroup>/<subvolume>`` parts for convenience
     provider
-        Optional. One of ``samba-vfs`` or ``kcephfs`` (``kcephfs`` is not yet
-        supported) . Selects how CephFS storage should be provided to the share
+        Optional. Selects how CephFS storage should be provided to the share.
+        The value may be one of ``samba-vfs``, ``samba-vfs/classic``,
+        ``samba-vfs/new`` or ``samba-vfs/proxied``. If unspecified
+        ``samba-vfs`` is assumed.
+
+        Selecting ``samba-vfs/new`` selects the new samba vfs plugin to connect to
+        cephfs and ``samba-vfs/proxied`` uses the new vfs plugin but routes the
+        connections through a proxy. Using the proxy allows for more number of
+        simultaneous client connections to the share but comes at the cost of
```

Comment:
"more number of" maybe should be "a greater number of" ?

Reply:
Thanks. Done.